app/utils/misc.js

```diff
@@ -51,6 +51,22 @@ export function getObjArrayKey(obj, key, defaultValue = '') {
 }
 
 /**
+ * get value from key or default if not set or not a string,
+ * also works for arrays
+ *
+ * @param {Object} obj Basic object or array
+ * @param {String} key Key to look for
+ * @param {*} defaultValue Default to return if key is not set
+ */
+export function getObjArrayKeyStringOnly(obj, key, defaultValue = '') {
```

Comment:
Could make this function more generic as like
```
export function getValidatedArrayKeyFromObject(obj, key, default, validator) {
  return obj &&
    Object.prototype.hasOwnProperty.call(obj, key) && validator(obj[key]) ?
      obj[key] : defaultValue;
}
```
which you'd call like
```
getValidatedArrayKeyFromObject({test: 'yep'}, 'test', false, 
  (value) => (value instanceof String)); // -> 'yep'

getValidatedArrayKeyFromObject({test: 12}, 'test', false, 
  (value) => (value instanceof String)); // -> false
```
Just a thought, the way you have it is fine 😄